app/graph/mutations/user_two_factor_authentication_mutation.rb

```diff
@@ -0,0 +1,21 @@
+UserTwoFactorAuthenticationMutation = GraphQL::Relay::Mutation.define do
+  name 'UserTwoFactorAuthentication'
+
+  input_field :id, !types.Int
+  input_field :password, !types.String
+  input_field :qrcode, types.String
+  input_field :opt_required, types.Boolean
+
+  return_field :success, types.Boolean
+  return_field :user, UserType
+
+  resolve -> (_root, inputs, _ctx) {
+    user = User.where(id: inputs[:id]).last
```

Comment:
Please be sure that this user is the same as the current_api_user - we don't one user to request the 2FA code for another user :)

Reply:
Fixed